R/annotate.R

```diff
@@ -1,42 +1,37 @@
 #' Function annotateDGEobj
```

Comment:
It would be nice to add logic to make it so that you can pass either a character string (file location) or a named list.  This would let users read in their file and modify it themselves and send us the results without having to write them to a file to be read back in. 

To do this let's change "annotationFile" to annotations and explain that this is either a character string file name  for a key/value pair list using equals as the separator or a **named list** of values.  For the list we need to check not only that it is a list but that it has names (keys) and provide appropriate messages.

Reply:
This is a fairly common construct in R, so I think the effort is worth the convenience.